Cargo.lock

```diff
@@ -55,18 +55,26 @@ dependencies = [
 
 [[package]]
 name = "aho-corasick"
-version = "0.6.8"
+version = "0.6.10"
```

Comment:
did you run `cargo update`? :)
it would make backporting harder, so please revert

Reply:
Hm, I just deleted the lock file and re-built the project...

Reply:
instead of deleting, can you checkout `Cargo.lock` from master and build it with it?

```
git checkout master Cargo.lock
cargo check
```

Reply:
Yeah sure, sorry about that. Done.